Userland/Libraries/LibC/netinet/in_systm.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2021, Leon Albrecht <leon2002.la@gmail.com>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#pragma once
+
+#include <bits/stdint.h>
+#include <sys/cdefs.h>
+
+__BEGIN_DECLS
+
+typedef uint16_t n_short;
+typedef uint32_t n_long;
+
+typedef uint32_t n_time;
+
+// BSD implements a `n_time iptime()` function for the kernel
+// to generate a n_time with the ms since GMT 00:00
```

Comment:
What is the purpose of this comment? Is it meant to be a FIXME?
Also `a n_time` -> `an n_time`.

Reply:
General information about the header and compatablity for future development, I know that BSD uses and implemetns it in the icmp_input function

OK thats an edgcase of a/an that I have never encountered before; will fix that

Reply:
with the typo you mean the second line?

Reply:
Both lines are typos. It's "an n_time", not "a n_time" :)

Reply:
This information doesn't seem relevant since it refers to an API we don't implement.
If this is something we should implement, we could have a comment saying e.g "FIXME: Implement iptime() like BSD"

Reply:
Then I'll remove it for now